### PR TITLE
Add harfbuzz

### PR DESCRIPTION
**Pull request by Mstrodl**

### CODE REVIEW

File: switch/harfbuzz/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+depends=('switch-freetype')
+
+build() {
+  mkdir harfbuzz-$pkgver/build
+  cd harfbuzz-$pkgver/build
+
+  source "${DEVKITPRO}/switchvars.sh"
+
+  cmake -G"Unix Makefiles" -DCMAKE_TOOLCHAIN_FILE="${DEVKITPRO}/switch.cmake" \
+    -DUSE_SWITCH=ON -DCMAKE_INSTALL_PREFIX="${PORTLIBS_PREFIX}"  \
```
Comment (carstene1ns) [inline switch/harfbuzz/PKGBUILD:23]: `USE_SWITCH` seems not to be used in harfbuzz' cmakery.

**Comment by carstene1ns** [comment]
Please be aware of the circular dependency with freetype package. This can have unwanted side effects if rebuilding either one.

File: switch/harfbuzz/PKGBUILD (hunk 1 above)
Comment (Mstrodl) [reply switch/harfbuzz/PKGBUILD:23]: Yep, sorry.

**Comment by Benau** [comment]
https://github.com/harfbuzz/harfbuzz/issues/2524#issuecomment-780957796

Even macosx homebrew doesn't rebuild freetype with harfbuzz, so i guess it should be ok

**Comment by rhaleblian** [comment]
Speak of the devil.
I was looking into getting harfbuzz for NDS, for dslibris.
I was in the process of stealing PKGBUILD from the switch subtree.
Pls advise on any tips etc regarding process.

**Comment by rhaleblian** [comment]
#266

**Pull request merged**